mm2src/coins/utxo/qtum.rs

```diff
@@ -903,7 +903,9 @@ impl MarketCoinOps for QtumCoin {
 impl MmCoin for QtumCoin {
     fn is_asset_chain(&self) -> bool { utxo_common::is_asset_chain(&self.utxo_arc) }
 
-    fn spawner(&self) -> CoinFutSpawner { CoinFutSpawner::new(&self.as_ref().abortable_system) }
+    fn get_ctx(&self) -> Option<MmArc> { MmArc::from_weak(&self.as_ref().ctx) }
```

Comment:
I'm not sure if I mentioned this before (since my last review was a month ago), but why are we implementing this at the trait level? Coins don't create MmArc instances—they're already available at higher levels and some coins just receive them during initialization. We could likely pass MmArc where needed, as we do for certain coins. Adding this function seems unnecessary and inconsistent across different coins.

Reply:
Why it's not in https://github.com/KomodoPlatform/komodo-defi-framework/blob/547fc2bb4c03980379cfd219995e1d5ffa199bde/mm2src/coins/utxo/utxo_tx_history_v2.rs#L145-L152 ?

Reply:
that's a good place as well. im down to storing it (streaming manager) there if u are keen on the coins to expose no `ctx` :(

Reply:
> into an unrelated trait

gonna have to disagree with that, but guess that's a topic for another day :)

addressed in [65bd662](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172/commits/65bd66253220e210d3b85354deb0ae6588010020) & [e65ace0](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172/commits/e65ace0b074567a5f003fc4cdf947a229475bbb2)